infra/Pos/Slotting/Impl/Simple.hs

```diff
@@ -47,21 +46,30 @@ mkSimpleSlottingVar = atomically $ newTVar $ SimpleSlottingState $ unflattenSlot
 -- Implementation
 ----------------------------------------------------------------------------
 
-getCurrentSlotSimple :: SimpleSlottingMode m => SimpleSlottingVar -> m (Maybe SlotId)
+getCurrentSlotSimple 
+    :: (SimpleSlottingMode m, MonadThrow m)
```

Comment:
Here I also think it would be better to add `MonadThrow` to `SimpleSlottingMode`.